public_html/wp-content/mu-plugins/blocks/view/shared/grid-layout.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace WordCamp\Blocks\Shared\Components;
+
+/**
+ * @since Blocks v1.0
+ * Provides rendering for grid-layout component.
+ *
+ * @param string $layout            Whether the layout is `grid` or `list`.
+ * @param int    $columns           Number of columns if layout is `grid`. Assumed to be 1 if layout is grid.
+ * @param array  $children          Array of posts to rendered inside layout. Should be output markup.
+ * @param array  $container_classes Array of classes that will be added to container.
+ *
+ * @return string Markup of output layout.
+ */
+function render_grid_layout( $layout, $columns, $children, $container_classes ) {
```

Comment:
@vedanshujain Typically the way we've done these in other parts of the codebase is to have the function that creates the variables live inside the normal `includes`/`classes`/etc folder, and then only the HTML itself in the `view/` folder. That keeps the logic and presentation separated. I wonder if we should do that here too, for consistency, and more of a pure MVC-ish layout?

Reply:
this file does not have too much of a view, so we can move to `/include` folder. I will do that, meanwhile feel free to use these files.

Reply:
This is being fixed in #66